### ISSUE TITLE
v1.6.0 package is broken

### ISSUE THREAD

**Issue by candalfigomoro** [conda-forge/neuralforecast-feedstock#14]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

neuralforecast 1.6.0 requires

`numpy>=1.21.6 pandas>=1.3.5 torch>=2.0.0 pytorch-lightning>=2.0.0 ray[tune]>=2.2.0` and python >= 3.8

see https://github.com/Nixtla/neuralforecast/blob/main/settings.ini#L18C16-L18C98

However the meta.yaml recipe for the conda package has not been updated, see https://github.com/conda-forge/neuralforecast-feedstock/blob/main/recipe/meta.yaml#L32

meta.yaml requirements should be something like:

```
requirements:
  host:
    - python >=3.8
    - pip
  run:
    - python >=3.8
    - numpy >=1.21.6
    - pandas >=1.3.5
    - pytorch >=2.0.0
    - pytorch-lightning >=2.0.0
    - ray-tune >=2.2.0
```

### Installed packages

```shell
_libgcc_mutex             0.1                 conda_forge    conda-forge
_openmp_mutex             4.5                       2_gnu    conda-forge
absl-py                   1.4.0                    pypi_0    pypi
aiohttp                   3.8.4                    pypi_0    pypi
aiohttp-cors              0.7.0                    pypi_0    pypi
aiorwlock                 1.3.0                    pypi_0    pypi
aiosignal                 1.3.1                    pypi_0    pypi
ansiwrap                  0.8.4                    pypi_0    pypi
anyio                     3.7.0              pyhd8ed1ab_1    conda-forge
argon2-cffi               21.3.0             pyhd8ed1ab_0    conda-forge
argon2-cffi-bindings      21.2.0          py310h5764c6d_3    conda-forge
asttokens                 2.2.1              pyhd8ed1ab_0    conda-forge
async-timeout             4.0.2                    pypi_0    pypi
attrs                     23.1.0             pyh71513ae_1    conda-forge
babel                     2.12.1                   pypi_0    pypi
backcall                  0.2.0              pyh9f0ad1d_0    conda-forge
backoff                   2.2.1                    pypi_0    pypi
backports                 1.0                pyhd8ed1ab_3    conda-forge
backports.functools_lru_cache 1.6.4              pyhd8ed1ab_0    conda-forge
beatrix-jupyterlab        2023.615.192330          pypi_0    pypi
beautifulsoup4            4.12.2             pyha770c72_0    conda-forge
blas                      2.16                        mkl    conda-forge
bleach                    6.0.0              pyhd8ed1ab_0    conda-forge
blessed                   1.20.0                   pypi_0    pypi
boltons                   23.0.0             pyhd8ed1ab_0    conda-forge
brotli                    1.0.9                h166bdaf_8    conda-forge
brotli-bin                1.0.9                h166bdaf_8    conda-forge
bzip2                     1.0.8                h7f98852_4    conda-forge
c-ares                    1.19.1               hd590300_0    conda-forge
ca-certificates           2023.5.7             hbcca054_0    conda-forge
cachetools                5.3.1                    pypi_0    pypi
certifi                   2023.5.7           pyhd8ed1ab_0    conda-forge
cffi                      1.15.1          py310h255011f_3    conda-forge
charset-normalizer        3.1.0              pyhd8ed1ab_0    conda-forge
click                     8.1.3                    pypi_0    pypi
cloud-tpu-client          0.10                     pypi_0    pypi
cloudpickle               2.2.1                    pypi_0    pypi
colorama                  0.4.6              pyhd8ed1ab_0    conda-forge
colorful                  0.5.5                    pypi_0    pypi
comm                      0.1.3              pyhd8ed1ab_0    conda-forge
conda                     23.5.0          py310hff52083_1    conda-forge
conda-package-handling    2.0.2              pyh38be061_0    conda-forge
conda-package-streaming   0.8.0              pyhd8ed1ab_0    conda-forge
contourpy                 1.1.0                    pypi_0    pypi
cryptography              41.0.1          py310h75e40e8_0    conda-forge
cuda-cudart               11.7.99                       0    nvidia
cuda-cupti                11.7.101                      0    nvidia
cuda-libraries            11.7.1                        0    nvidia
cuda-nvrtc                11.7.99                       0    nvidia
cuda-nvtx                 11.7.91                       0    nvidia
cuda-runtime              11.7.1                        0    nvidia
cycler                    0.11.0                   pypi_0    pypi
cython                    0.29.35                  pypi_0    pypi
dacite                    1.8.1                    pypi_0    pypi
db-dtypes                 1.1.1                    pypi_0    pypi
debugpy                   1.6.7           py310heca2aa9_0    conda-forge
decorator                 5.1.1              pyhd8ed1ab_0    conda-forge
defusedxml                0.7.1              pyhd8ed1ab_0    conda-forge
deprecated                1.2.14                   pypi_0    pypi
distlib                   0.3.6                    pypi_0    pypi
dlenv-base                1.0.20230616            py310_0    file:///tmp/conda-pkgs
dm-tree                   0.1.8                    pypi_0    pypi
docker                    6.1.3                    pypi_0    pypi
docstring-parser          0.15                     pypi_0    pypi
entrypoints               0.4                pyhd8ed1ab_0    conda-forge
exceptiongroup            1.1.1              pyhd8ed1ab_0    conda-forge
executing                 1.2.0              pyhd8ed1ab_0    conda-forge
fastapi                   0.97.0                   pypi_0    pypi
filelock                  3.12.2             pyhd8ed1ab_0    conda-forge
fire                      0.5.0                    pypi_0    pypi
flit-core                 3.9.0              pyhd8ed1ab_0    conda-forge
fmt                       9.1.0                h924138e_0    conda-forge
fonttools                 4.40.0                   pypi_0    pypi
frozenlist                1.3.3                    pypi_0    pypi
fsspec                    2023.6.0                 pypi_0    pypi
gcsfs                     2023.6.0                 pypi_0    pypi
gitdb                     4.0.10                   pypi_0    pypi
gitpython                 3.1.31                   pypi_0    pypi
gmp                       6.2.1                h58526e2_0    conda-forge
gmpy2                     2.1.2           py310h3ec546c_1    conda-forge
google-api-core           1.34.0                   pypi_0    pypi
google-api-python-client  1.8.0                    pypi_0    pypi
google-auth               2.20.0                   pypi_0    pypi
google-auth-httplib2      0.1.0                    pypi_0    pypi
google-auth-oauthlib      1.0.0                    pypi_0    pypi
google-cloud-aiplatform   1.26.0                   pypi_0    pypi
google-cloud-artifact-registry 1.8.1                    pypi_0    pypi
google-cloud-bigquery     3.11.1                   pypi_0    pypi
google-cloud-bigquery-storage 2.20.0                   pypi_0    pypi
google-cloud-core         2.3.2                    pypi_0    pypi
google-cloud-datastore    1.15.5                   pypi_0    pypi
google-cloud-language     2.10.0                   pypi_0    pypi
google-cloud-monitoring   2.15.0                   pypi_0    pypi
google-cloud-resource-manager 1.10.1                   pypi_0    pypi
google-cloud-storage      2.9.0                    pypi_0    pypi
google-crc32c             1.5.0                    pypi_0    pypi
google-resumable-media    2.5.0                    pypi_0    pypi
googleapis-common-protos  1.59.1                   pypi_0    pypi
gpustat                   1.0.0                    pypi_0    pypi
greenlet                  2.0.2                    pypi_0    pypi
grpc-google-iam-v1        0.12.6                   pypi_0    pypi
grpcio                    1.51.3                   pypi_0    pypi
grpcio-status             1.48.2                   pypi_0    pypi
gymnasium                 0.26.3                   pypi_0    pypi
gymnasium-notices         0.0.1                    pypi_0    pypi
h11                       0.14.0                   pypi_0    pypi
htmlmin                   0.1.12                   pypi_0    pypi
httplib2                  0.22.0                   pypi_0    pypi
icu                       72.1                 hcb278e6_0    conda-forge
idna                      3.4                pyhd8ed1ab_0    conda-forge
imagehash                 4.3.1                    pypi_0    pypi
imageio                   2.31.1                   pypi_0    pypi
importlib-metadata        6.0.1                    pypi_0    pypi
importlib_metadata        6.6.0                hd8ed1ab_0    conda-forge
importlib_resources       5.12.0             pyhd8ed1ab_0    conda-forge
intel-openmp              2023.1.0         hdb19cb5_46305  
ipykernel                 6.23.2                   pypi_0    pypi
ipython                   8.14.0             pyh41d4057_0    conda-forge
ipython-genutils          0.2.0                    pypi_0    pypi
ipython-sql               0.5.0                    pypi_0    pypi
ipython_genutils          0.2.0                      py_1    conda-forge
ipywidgets                8.0.6                    pypi_0    pypi
jaraco-classes            3.2.3                    pypi_0    pypi
jedi                      0.18.2             pyhd8ed1ab_0    conda-forge
jeepney                   0.8.0                    pypi_0    pypi
jinja2                    3.1.2              pyhd8ed1ab_1    conda-forge
joblib                    1.2.0                    pypi_0    pypi
json5                     0.9.14                   pypi_0    pypi
jsonpatch                 1.32               pyhd8ed1ab_0    conda-forge
jsonpointer               2.0                        py_0    conda-forge
jsonschema                4.17.3             pyhd8ed1ab_0    conda-forge
jupyter-client            7.4.9                    pypi_0    pypi
jupyter-http-over-ws      0.0.8                    pypi_0    pypi
jupyter-server            1.24.0                   pypi_0    pypi
jupyter-server-mathjax    0.2.6                    pypi_0    pypi
jupyter-server-proxy      4.0.0                    pypi_0    pypi
jupyter_client            8.2.0              pyhd8ed1ab_0    conda-forge
jupyter_core              5.3.1           py310hff52083_0    conda-forge
jupyter_events            0.6.3              pyhd8ed1ab_0    conda-forge
jupyter_server            2.6.0              pyhd8ed1ab_0    conda-forge
jupyter_server_terminals  0.4.4              pyhd8ed1ab_1    conda-forge
jupyterlab                3.4.8                    pypi_0    pypi
jupyterlab-git            0.41.0                   pypi_0    pypi
jupyterlab-server         2.23.0                   pypi_0    pypi
jupyterlab-widgets        3.0.7                    pypi_0    pypi
jupyterlab_pygments       0.2.2              pyhd8ed1ab_0    conda-forge
jupytext                  1.14.6                   pypi_0    pypi
keyring                   23.13.1                  pypi_0    pypi
keyrings-google-artifactregistry-auth 1.1.2                    pypi_0    pypi
keyutils                  1.6.1                h166bdaf_0    conda-forge
kfp                       1.8.22                   pypi_0    pypi
kfp-pipeline-spec         0.1.16                   pypi_0    pypi
kfp-server-api            1.8.5                    pypi_0    pypi
kiwisolver                1.4.4                    pypi_0    pypi
krb5                      1.21.1               h659d440_0    conda-forge
kubernetes                25.3.0                   pypi_0    pypi
lazy-loader               0.2                      pypi_0    pypi
ld_impl_linux-64          2.40                 h41732ed_0    conda-forge
libarchive                3.6.2                h039dbb9_1    conda-forge
libblas                   3.8.0                    16_mkl    conda-forge
libbrotlicommon           1.0.9                h166bdaf_8    conda-forge
libbrotlidec              1.0.9                h166bdaf_8    conda-forge
libbrotlienc              1.0.9                h166bdaf_8    conda-forge
libcblas                  3.8.0                    16_mkl    conda-forge
libcublas                 11.10.3.66                    0    nvidia
libcufft                  10.7.2.124           h4fbf590_0    nvidia
libcufile                 1.7.0.149                     0    nvidia
libcurand                 10.3.3.53                     0    nvidia
libcurl                   8.1.2                hca28451_1    conda-forge
libcusolver               11.4.0.1                      0    nvidia
libcusparse               11.7.4.91                     0    nvidia
libedit                   3.1.20191231         he28a2e2_2    conda-forge
libev                     4.33                 h516909a_1    conda-forge
libffi                    3.4.2                h7f98852_5    conda-forge
libgcc-ng                 13.1.0               he5830b7_0    conda-forge
libgfortran-ng            7.5.0               h14aa051_20    conda-forge
libgfortran4              7.5.0               h14aa051_20    conda-forge
libgomp                   13.1.0               he5830b7_0    conda-forge
libiconv                  1.17                 h166bdaf_0    conda-forge
liblapack                 3.8.0                    16_mkl    conda-forge
liblapacke                3.8.0                    16_mkl    conda-forge
libmamba                  1.4.7                h658169a_0    conda-forge
libmambapy                1.4.7           py310h8aae740_0    conda-forge
libnghttp2                1.52.0               h61bc06f_0    conda-forge
libnpp                    11.7.4.75                     0    nvidia
libnsl                    2.0.0                h7f98852_0    conda-forge
libnvjpeg                 11.8.0.2                      0    nvidia
libsodium                 1.0.18               h36c2ea0_1    conda-forge
libsolv                   0.7.24               hfc55251_1    conda-forge
libsqlite                 3.42.0               h2797004_0    conda-forge
libssh2                   1.11.0               h0841786_0    conda-forge
libstdcxx-ng              13.1.0               hfd8a6a1_0    conda-forge
libuuid                   2.38.1               h0b41bf4_0    conda-forge
libuv                     1.44.2               h166bdaf_0    conda-forge
libxml2                   2.11.4               h0d562d8_0    conda-forge
libzlib                   1.2.13               hd590300_5    conda-forge
llvmlite                  0.40.1rc1                pypi_0    pypi
lz4                       4.3.2                    pypi_0    pypi
lz4-c                     1.9.4                hcb278e6_0    conda-forge
lzo                       2.10              h516909a_1000    conda-forge
mamba                     1.4.7           py310h51d5547_0    conda-forge
markdown-it-py            2.2.0                    pypi_0    pypi
markupsafe                2.1.3           py310h2372a71_0    conda-forge
matplotlib                3.7.1                    pypi_0    pypi
matplotlib-inline         0.1.6              pyhd8ed1ab_0    conda-forge
mdit-py-plugins           0.4.0                    pypi_0    pypi
mdurl                     0.1.2                    pypi_0    pypi
mistune                   2.0.5              pyhd8ed1ab_0    conda-forge
mkl                       2020.2                      256  
more-itertools            9.1.0                    pypi_0    pypi
mpc                       1.3.1                hfe3b2da_0    conda-forge
mpfr                      4.2.0                hb012696_0    conda-forge
mpmath                    1.3.0              pyhd8ed1ab_0    conda-forge
msgpack                   1.0.5                    pypi_0    pypi
multidict                 6.0.4                    pypi_0    pypi
multimethod               1.9.1                    pypi_0    pypi
nb_conda                  2.2.1                    unix_6    conda-forge
nb_conda_kernels          2.3.1           py310hff52083_2    conda-forge
nbclassic                 1.0.0              pyhb4ecaf3_1    conda-forge
nbclient                  0.8.0              pyhd8ed1ab_0    conda-forge
nbconvert                 7.5.0              pyhd8ed1ab_0    conda-forge
nbconvert-core            7.5.0              pyhd8ed1ab_0    conda-forge
nbconvert-pandoc          7.5.0              pyhd8ed1ab_0    conda-forge
nbdime                    3.2.0                    pypi_0    pypi
nbformat                  5.9.0              pyhd8ed1ab_0    conda-forge
ncurses                   6.4                  hcb278e6_0    conda-forge
nest-asyncio              1.5.6              pyhd8ed1ab_0    conda-forge
networkx                  3.1                pyhd8ed1ab_0    conda-forge
nodejs                    18.15.0              h4abf6b9_1    conda-forge
notebook                  6.5.4              pyha770c72_0    conda-forge
notebook-executor         0.2                      pypi_0    pypi
notebook-shim             0.2.3              pyhd8ed1ab_0    conda-forge
numba                     0.57.0                   pypi_0    pypi
numpy                     1.23.5                   pypi_0    pypi
nvidia-ml-py              11.495.46                pypi_0    pypi
oauth2client              4.1.3                    pypi_0    pypi
oauthlib                  3.2.2                    pypi_0    pypi
opencensus                0.11.2                   pypi_0    pypi
opencensus-context        0.1.3                    pypi_0    pypi
openssl                   3.1.1                hd590300_1    conda-forge
opentelemetry-api         1.18.0                   pypi_0    pypi
opentelemetry-exporter-otlp 1.18.0                   pypi_0    pypi
opentelemetry-exporter-otlp-proto-common 1.18.0                   pypi_0    pypi
opentelemetry-exporter-otlp-proto-grpc 1.18.0                   pypi_0    pypi
opentelemetry-exporter-otlp-proto-http 1.18.0                   pypi_0    pypi
opentelemetry-proto       1.18.0                   pypi_0    pypi
opentelemetry-sdk         1.18.0                   pypi_0    pypi
opentelemetry-semantic-conventions 0.39b0                   pypi_0    pypi
overrides                 7.3.1              pyhd8ed1ab_0    conda-forge
packaging                 23.1               pyhd8ed1ab_0    conda-forge
pandas                    1.5.3                    pypi_0    pypi
pandas-profiling          3.6.6                    pypi_0    pypi
pandoc                    3.1.3                h32600fe_0    conda-forge
pandocfilters             1.5.0              pyhd8ed1ab_0    conda-forge
papermill                 2.4.0                    pypi_0    pypi
parso                     0.8.3              pyhd8ed1ab_0    conda-forge
patsy                     0.5.3                    pypi_0    pypi
pexpect                   4.8.0              pyh1a96a4e_2    conda-forge
phik                      0.12.3                   pypi_0    pypi
pickleshare               0.7.5                   py_1003    conda-forge
pillow                    9.5.0                    pypi_0    pypi
pip                       23.1.2             pyhd8ed1ab_0    conda-forge
pkgutil-resolve-name      1.3.10             pyhd8ed1ab_0    conda-forge
platformdirs              3.5.3              pyhd8ed1ab_0    conda-forge
plotly                    5.15.0                   pypi_0    pypi
pluggy                    1.0.0              pyhd8ed1ab_5    conda-forge
prettytable               3.8.0                    pypi_0    pypi
prometheus_client         0.17.0             pyhd8ed1ab_0    conda-forge
prompt-toolkit            3.0.38             pyha770c72_0    conda-forge
prompt_toolkit            3.0.38               hd8ed1ab_0    conda-forge
proto-plus                1.22.2                   pypi_0    pypi
protobuf                  3.20.3                   pypi_0    pypi
psutil                    5.9.3                    pypi_0    pypi
ptyprocess                0.7.0              pyhd3deb0d_0    conda-forge
pure_eval                 0.2.2              pyhd8ed1ab_0    conda-forge
py-spy                    0.3.14                   pypi_0    pypi
pyarrow                   12.0.1                   pypi_0    pypi
pyasn1                    0.5.0                    pypi_0    pypi
pyasn1-modules            0.3.0                    pypi_0    pypi
pybind11-abi              4                    hd8ed1ab_3    conda-forge
pycosat                   0.6.4           py310h5764c6d_1    conda-forge
pycparser                 2.21               pyhd8ed1ab_0    conda-forge
pydantic                  1.10.9                   pypi_0    pypi
pygments                  2.15.1             pyhd8ed1ab_0    conda-forge
pyjwt                     2.7.0                    pypi_0    pypi
pyopenssl                 23.2.0             pyhd8ed1ab_1    conda-forge
pyparsing                 3.0.9                    pypi_0    pypi
pyrsistent                0.19.3          py310h1fa729e_0    conda-forge
pysocks                   1.7.1              pyha2e5f31_6    conda-forge
python                    3.10.11         he550d4f_0_cpython    conda-forge
python-dateutil           2.8.2              pyhd8ed1ab_0    conda-forge
python-fastjsonschema     2.17.1             pyhd8ed1ab_0    conda-forge
python-json-logger        2.0.7              pyhd8ed1ab_0    conda-forge
python_abi                3.10                    3_cp310    conda-forge
pytorch                   2.0.1           py3.10_cuda11.7_cudnn8.5.0_0    pytorch
pytorch-cuda              11.7                 h778d358_5    pytorch
pytorch-mutex             1.0                        cuda    pytorch
pytz                      2023.3                   pypi_0    pypi
pywavelets                1.4.1                    pypi_0    pypi
pyyaml                    6.0             py310h5764c6d_5    conda-forge
pyzmq                     25.1.0          py310h5bbb5d0_0    conda-forge
ray                       2.5.0                    pypi_0    pypi
ray-cpp                   2.5.0                    pypi_0    pypi
readline                  8.2                  h8228510_1    conda-forge
reproc                    14.2.4               h0b41bf4_0    conda-forge
reproc-cpp                14.2.4               hcb278e6_0    conda-forge
requests                  2.31.0             pyhd8ed1ab_0    conda-forge
requests-oauthlib         1.3.1                    pypi_0    pypi
requests-toolbelt         0.10.1                   pypi_0    pypi
retrying                  1.3.4                    pypi_0    pypi
rfc3339-validator         0.1.4              pyhd8ed1ab_0    conda-forge
rfc3986-validator         0.1.1              pyh9f0ad1d_0    conda-forge
rich                      13.4.2                   pypi_0    pypi
ruamel.yaml               0.17.31         py310h2372a71_0    conda-forge
ruamel.yaml.clib          0.2.7           py310h1fa729e_1    conda-forge
scikit-image              0.21.0                   pypi_0    pypi
scikit-learn              1.2.2                    pypi_0    pypi
scipy                     1.10.1                   pypi_0    pypi
seaborn                   0.12.2                   pypi_0    pypi
secretstorage             3.3.3                    pypi_0    pypi
send2trash                1.8.2              pyh41d4057_0    conda-forge
setuptools                67.7.2             pyhd8ed1ab_0    conda-forge
shapely                   1.8.5.post1              pypi_0    pypi
simpervisor               1.0.0                    pypi_0    pypi
six                       1.16.0             pyh6c4a22f_0    conda-forge
smart-open                6.3.0                    pypi_0    pypi
smmap                     5.0.0                    pypi_0    pypi
sniffio                   1.3.0              pyhd8ed1ab_0    conda-forge
soupsieve                 2.4.1                    pypi_0    pypi
sqlalchemy                2.0.16                   pypi_0    pypi
sqlparse                  0.4.4                    pypi_0    pypi
stack_data                0.6.2              pyhd8ed1ab_0    conda-forge
starlette                 0.27.0                   pypi_0    pypi
statsmodels               0.14.0                   pypi_0    pypi
strip-hints               0.1.10                   pypi_0    pypi
sympy                     1.12            pypyh9d50eac_103    conda-forge
tabulate                  0.9.0                    pypi_0    pypi
tangled-up-in-unicode     0.2.0                    pypi_0    pypi
tenacity                  8.2.2                    pypi_0    pypi
tensorboardx              2.6                      pypi_0    pypi
termcolor                 2.3.0                    pypi_0    pypi
terminado                 0.17.1             pyh41d4057_0    conda-forge
textwrap3                 0.9.2                    pypi_0    pypi
threadpoolctl             3.1.0                    pypi_0    pypi
tifffile                  2023.4.12                pypi_0    pypi
tinycss2                  1.2.1              pyhd8ed1ab_0    conda-forge
tk                        8.6.12               h27826a3_0    conda-forge
toml                      0.10.2                   pypi_0    pypi
tomli                     2.0.1                    pypi_0    pypi
toolz                     0.12.0             pyhd8ed1ab_0    conda-forge
torchtriton               2.0.0                     py310    pytorch
tornado                   6.3.2           py310h2372a71_0    conda-forge
tqdm                      4.65.0             pyhd8ed1ab_1    conda-forge
traitlets                 5.9.0              pyhd8ed1ab_0    conda-forge
typeguard                 2.13.3                   pypi_0    pypi
typer                     0.9.0                    pypi_0    pypi
typing-extensions         4.6.3                hd8ed1ab_0    conda-forge
typing_extensions         4.6.3              pyha770c72_0    conda-forge
typing_utils              0.1.0              pyhd8ed1ab_0    conda-forge
tzdata                    2023c                h71feb2d_0    conda-forge
uritemplate               3.0.1                    pypi_0    pypi
urllib3                   1.26.16                  pypi_0    pypi
uvicorn                   0.22.0                   pypi_0    pypi
virtualenv                20.21.0                  pypi_0    pypi
visions                   0.7.5                    pypi_0    pypi
wcwidth                   0.2.6              pyhd8ed1ab_0    conda-forge
webencodings              0.5.1                    pypi_0    pypi
websocket-client          1.5.3              pyhd8ed1ab_0    conda-forge
wheel                     0.40.0             pyhd8ed1ab_0    conda-forge
widgetsnbextension        4.0.7                    pypi_0    pypi
wordcloud                 1.9.2                    pypi_0    pypi
wrapt                     1.15.0                   pypi_0    pypi
xz                        5.2.6                h166bdaf_0    conda-forge
yaml                      0.2.5                h7f98852_2    conda-forge
yaml-cpp                  0.7.0                h27087fc_2    conda-forge
yarl                      1.9.2                    pypi_0    pypi
ydata-profiling           4.2.0                    pypi_0    pypi
zeromq                    4.3.4                h9c3ff4c_1    conda-forge
zipp                      3.15.0             pyhd8ed1ab_0    conda-forge
zlib                      1.2.13               hd590300_5    conda-forge
zstandard                 0.19.0          py310hdeb6495_1    conda-forge
zstd                      1.5.2                h3eb15da_6    conda-forge
```

### Environment info

```shell
active environment : base
    active env location : /opt/conda
            shell level : 1
       user config file : /root/.condarc
 populated config files : /opt/conda/.condarc
          conda version : 23.5.0
    conda-build version : not installed
         python version : 3.10.11.final.0
       virtual packages : __archspec=1=x86_64
                          __cuda=11.6=0
                          __glibc=2.31=0
                          __linux=5.10.0=0
                          __unix=0=0
       base environment : /opt/conda  (writable)
      conda av data dir : /opt/conda/etc/conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /opt/conda/pkgs
                          /root/.conda/pkgs
       envs directories : /opt/conda/envs
                          /root/.conda/envs
               platform : linux-64
             user-agent : conda/23.5.0 requests/2.31.0 CPython/3.10.11 Linux/5.10.0-23-cloud-amd64 debian/11 glibc/2.31
                UID:GID : 0:0
             netrc file : None
           offline mode : False
```

**Comment by candalfigomoro** [comment]
@FedericoGarza

**Comment by FedericoGarza** [comment]
Thank you, @candalfigomoro!

Fixed in #15. :)﻿
